Tests/PostgreSQLTests/PostgreSQLTests.swift

```diff
@@ -103,14 +103,15 @@ extension Album: ModelProtocol {
 // MARK: - Tests
 
 public class PostgreSQLTests: XCTestCase {
-    let connection = try! PostgreSQL.Connection(info: .init(URL(string: "postgres://localhost:5432/swift_test")!))
+    var connection: Connection!
 
     let logger = Logger(name: "SQL Logger", appenders: [StandardOutputAppender()])
 
     override public func setUp() {
         super.setUp()
-
+        
         do {
+            connection = try! PostgreSQL.Connection(info: .init(URL(string: "postgres://localhost:5432/swift_test")!))
```

Comment:
For tests each test should use it's own connection, we shouldn't share it due to busy states/dangling etc… 

Connections are also unrecoverable ( you can't "reconnect" )